src/lib/pages/account-details/data.ts

```diff
@@ -34,13 +36,29 @@ interface AccountDetailsTableCounts {
 export const useAccountDetailsTableCounts = (
   address: BechAddr
 ): AccountDetailsTableCounts => {
+  const isFullTier = useTierConfig() === "full";
   const {
     data,
     refetch,
     isLoading: isLoadingAccountTableCounts,
-  } = useAccountTableCounts(address);
+  } = useAccountTableCounts(address, { enabled: isFullTier });
   const { data: balances, isLoading: isBalancesLoading } = useBalances(address);
 
+  if (!isFullTier) {
+    return {
+      tableCounts: {
+        codesCount: undefined,
+        contractsAdminCount: undefined,
+        contractsCount: undefined,
+        txsCount: undefined,
```

Comment:
can we use instatiated contract by address LCD hook to get the count?
also for txs does `pagination.count_total` work? or too slow

Reply:
Most of the chain doesn't support count_total on instantiated contracts by address as follows [link](https://terra-testnet-api.polkachu.com/cosmwasm/wasm/v1/contracts/creator/terra1vgm7t8q43sfg0xq9693ah0xvgxv0qnkm9xtr54?pagination.count_total=true)